src/App.css

```diff
@@ -1,7 +1,17 @@
 .App {
   text-align: center;
+  margin-right: 45px;
 }
 
```

Comment:
Import ones: 
- Accessibility! hover and focus state for anything likable 
- flex- basis and flex for images to which when app mounts appear.
- Using border and background color with contrast to font-color to help them pop a bit more
- for individual meme page border for the main content,  using spacing, flexbox and maybe storing them in a container to visually indicate that they are part of same form and content. 
- Maybe using some text-shadow can help with a nice visual effect. 
- A lighter blue background can be easier on the eye. 
- using wrapper and setup snippet. 
- Border Radius can be very helpful